src/components/analyticVisualization/VisualizationGroup/VisualizationGroup.js

```diff
@@ -26,10 +27,38 @@ const findDefaultGroup = (type, groupList, settings) => {
     return defaultGroup
 }
 
-export const VisualizationGroup = ({ settings, onChange, groupList, type }) => {
-    const { data: id } = useSystemId()
+const isDefaultGroup = group => group.name === 'default'
```

Comment:
NIT: probably better to use `const isDefaultGroup = group => group.name === DEFAULT` as you've defined the constant already